plugin/pr-sanitizer/main.go

```diff
@@ -1,35 +1,18 @@
 package main
 
 import (
-	"flag"
-	"syscall"
-	"os/signal"
-	"net/url"
-	"strconv"
-	"net/http"
-	"golang.org/x/oauth2"
-	"context"
-
 	"github.com/sirupsen/logrus"
 	"github.com/google/go-github/github"
-	"k8s.io/test-infra/prow/pluginhelp/externalplugins"
-	"k8s.io/test-infra/prow/plugins"
 	"k8s.io/test-infra/prow/pluginhelp"
 
 	"github.com/arquillian/ike-prow-plugins/plugin/server"
-	. "github.com/arquillian/ike-prow-plugins/plugin/utils"
+	commonPlugin "github.com/arquillian/ike-prow-plugins/plugin"
```

Comment:
I would call it `pluginBootstrap` or something. `common` is a bit too ambiguous.